mlflow/server/js/src/experiment-tracking/components/ParallelCoordinatesPlotControls.js

```diff
@@ -26,13 +30,30 @@ export class ParallelCoordinatesPlotControls extends React.Component {
       selectedMetricKeys,
       handleParamsSelectChange,
       handleMetricsSelectChange,
+      onClearParamsSelect,
     } = this.props;
     return (
       <div className='plot-controls'>
         <div>
-          <FormattedMessage
-            defaultMessage='Parameters:'
-            description='Label text for parameters in parallel coordinates plot in MLflow'
+          <FlexBar
+            left={
+              <Spacer size='small' direction='horizontal'>
+                <FormattedMessage
+                  defaultMessage='Parameters:'
+                  description='Label text for parameters in parallel coordinates plot in MLflow'
+                />
+              </Spacer>
+            }
+            right={
+              <Spacer size='small' direction='horizontal'>
+                <Button dataTestId='clear-button' onClick={onClearParamsSelect}>
+                  <FormattedMessage
+                    defaultMessage='Clear All'
+                    description='String for the clear button to clear any selected parameters'
+                  />
+                </Button>
+              </Spacer>
```

Comment:
How does this button look like? Can you take a screenshot and upload it here?

Reply:
<img width="1609" alt="Screen Shot 2022-01-26 at 4 25 43 PM" src="https://user-images.githubusercontent.com/7851093/151269206-731c0981-2c28-45aa-8696-1c1ecc89a3f2.png">